Envoy.blade.php

```diff
@@ -264,11 +264,13 @@
 @endtask
 
 @task('cache_remote_release', ['on' => $remote_server])
+shopt -s expand_aliases
+source ~/.bashrc
 echo "Caching configs...";
-php73 artisan config:cache
+php73 {{ $source_dir }}/ {{ $release_dir }}/{{ $release }}/artisan config:cache
```

Comment:
This has a space in between the {{ $source_dir }}/ and {{ $release_dir }}

Reply:
Thanks for noticing! I copied it from https://github.com/waynestate/base-site/blob/cfe9ac46b6a64217ea8a5da4fad7a3a1ff3f5159/Envoy.blade.php#L260

Seems like it works still, but I will fix all instances of the extra space